docs/develop/evm/introduction.md

```diff
@@ -0,0 +1,19 @@
+---
+id: introduction
+title: Project Aurora Introduction
+sidebar_label: Introduction
+---
+
+[Project Aurora](https://aurora.dev/) delivers an Ethereum Virtual Machine
+(EVM) on the NEAR protocol. It is comprised of the following components:
+
+| Name                                                                                    | Description                            |
+| ----------------------------------------------------------------------------------------| -------------------------------------- |
+| **[Aurora Engine](https://github.com/aurora-is-near/aurora-engine)**                    | Implements an EVM on the NEAR protocol |
+| **[Aurora CLI](https://github.com/aurora-is-near/aurora-cli)**                          | Command-Line Interface (CLI) for the Aurora Engine |
+| **[Aurora Engine Client for JS](https://github.com/aurora-is-near/aurora.js)**          | JavaScript Client for the Aurora Engine |
```

Comment:
Let's use consistent title casing in the Description column. Right now it's title case for e.g. _"JavaScript Client for the Aurora Engine"_ but non-title case for _"Web3 provider to connect Ethereum frontend or Truffle to NEAR Protocol"_